servant-hmac-auth.cabal

```diff
@@ -88,16 +88,3 @@ test-suite servant-hmac-auth-test
   main-is:             Spec.hs
   ghc-options:         -threaded -rtsopts -with-rtsopts=-N
   build-depends:       servant-hmac-auth
-
-executable readme
-  import:              common-options
-  main-is:             README.lhs
-  build-depends:       aeson ^>= 1.4
-                     , http-client
-                     , servant
-                     , servant-hmac-auth
-                     , servant-client
-                     , servant-server
-                     , warp ^>= 3.3.5
-  build-tool-depends:  markdown-unlit:markdown-unlit
-  ghc-options:         -pgmL markdown-unlit
```

Comment:
is there a problem with supporting README.lhs? I think it would be beneficial to keep this executable.

Reply:
Sorry, I imagine it didn't build. I can get that building again.

Reply:
it's not the highest priority. I was mainly just curious. We can even make it work after 0.18 support is out. This will be up to @kahlil29 to make call on.